docs/expectation_gallery/azure-pipelines-expectation-gallery.yml

```diff
@@ -243,6 +770,70 @@ stages:
               targetFolder: 'static/gallery/'
               filesAcl: 'public-read'
 
+
+# Don't do build_package_gallery.py stuff...
+#   - Currently, this script will call the run_diagnostics method on every Expectation
+#     that exists in each package for all backends (even though this was already done
+#     by the build_gallery.py script)
+#   - The build_gallery.py script is now package aware
+#   - The Expectation Gallery site can now accept a URL that has a filter applied to
+#     limit results to a specific package, so no need to bundle them in the "packages" tab
```

Comment:
Should this commented out code and `build_package_gallery.py` be removed based on this comment?

Reply:
You should come to the Tuesday meeting next week. In a soon-ish later ticket, I'm going to make that thing not run any diagnostics or include the expectations in the package_manifests.json file.

For now, we just want whatever existing old package_manifests.json file on s3 to stay there, since the change will need to be coordinated with some frontend changes.

After some frontend changes that haven't been merged yet, we can link to the Expectations page with a specific filter applied (say for a particular package). https://deploy-preview-230--staging-great-expectations.netlify.app/expectations/?filterType=Package&viewType=Summary&showFilters=true&subFilterValues=capitalone_dataprofiler_expectations

Reply:
I don't understand - are you saying that this code will be re-enabled at that point?

Reply:
Re-enabled once it doesn't calculate the expectation_diagnostics or include expectations in it's output JSON. There is still all that unchanging metadata about the packages in the package_info.yml files (like https://github.com/great-expectations/great_expectations/blob/develop/contrib/capitalone_dataprofiler_expectations/package_info.yml).